src/Resources/views/bootstrap_4_layout.html.twig

```diff
@@ -7,11 +7,11 @@
             {% set locale = translationsFields.vars.name %}
 
             <li class="nav-item">
-                <a href="#{{ translationsFields.vars.id }}_a2lix_translations-fields" class="nav-link {% if app.request.locale == locale %}active{% endif %}" data-toggle="tab" role="tab">
+                <div data-bs-target="#{{ translationsFields.vars.id }}_a2lix_translations-fields" class="nav-link {% if app.request.locale == locale %}active{% endif %}" data-toggle="tab" role="tab">
```

Comment:
[data-target](https://getbootstrap.com/docs/4.6/components/navs/#using-data-attributes) for v4